static/js/test_util.ts

```diff
@@ -178,9 +178,11 @@ export const withFakeLocation = async (
   cb: () => Promise<void> | void
```

Comment:
This function runs a callback `cb` using a fake `window.location` property. See comment above function. Mocking window.location is a pain with JSDom.

But... When I first wrote it... I did it wrong. The callback never ever ran. Now it does.

There should have been no `catch`. Only `try { ... } finally { ... }`.